### PR TITLE
Update links that doesn't works

### PR DESCRIPTION
**Pull request by EmilieAig**
Update links that doesn’t works (when we click we obtain ‘not found’ or ‘404 error’), modifications on pages:
* Data Science
* Visualization
* Debugging & Profiling

### CODE REVIEW

File: .DS_Store
Comment (josephsalmon) [inline .DS_Store:1]: remove this file

File: Courses/.DS_Store
Comment (josephsalmon) [inline Courses/.DS_Store:1]: remove this file

File: Courses/Git/tp.quarto_ipynb
Comment (josephsalmon) [inline Courses/Git/tp.quarto_ipynb:1]: remove this file

**Review by josephsalmon — CHANGES_REQUESTED** [review]
Thanks for the feedabck. There are few files that need to be erase but otherwise LGTM.

File: .DS_Store
Comment (EmilieAig) [reply .DS_Store:1]: Thank you, it's done

File: Courses/.DS_Store
Comment (EmilieAig) [reply Courses/.DS_Store:1]: Thank you, it's done

**Review by josephsalmon — APPROVED** [review]
All good then.
I am validating the 1 pt bonus. Thanks again.

**Pull request merged**